AlertaDengue/upload/sinan/utils.py

```diff
@@ -80,7 +80,8 @@ def convert_date(col: Union[pd.Timestamp, str, None]) -> Optional[dt.date]:
             return None
         if isinstance(col, dt.date):
             return col
-        return pd.to_datetime(col, dayfirst=True).date()
+        # return pd.to_datetime(col, dayfirst=True).date()
```

Comment:
Removing this line for the rollback process, then it should be replaced for a more robust date parsing